### DISCUSSION TITLE
Are there keyboard shortcuts to delete everything before or after the cursor?

### DISCUSSION THREAD

**Discussion by legate** [microsoft/terminal#16179] [Q&A] [answered]
In Bash on Linux, Ctrl+U and Ctrl+K delete everything before and after the cursor respectively. Are there equivalent shortcuts for Windows Terminal? Inputting those shortcuts just makes `^U` and `^K` appear on the command line.

**Comment by elsaco** [comment, marked as answer]
@legate those are functions of the shell! If you're using powershell try running `Get-PSReadLineKeyHandler` and list the basic editing functions supported. There's `Ctrl+Home` and `Ctrl+End` doing the same `Ctrl-U` and `Ctrl-K` do in bash.
